spec/partials/line-break-grammar-spec.coffee

```diff
@@ -18,17 +18,16 @@ describe 'Should tokenize line break when', ->
 
   it 'contains simple character', ->
     tokens = grammar.tokenizeLines '''
-       +
       Foo +
+      Bar
```

Comment:
Yep, the sample is asciidoctor.rb is incorrect (in both cases).

While those examples are usually correct, they aren't gospel :)